easybuild/easyconfigs/g/GLFW/GLFW-3.4-GCCcore-12.2.0.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'GLFW'
+version = '3.4'
+
+homepage = 'https://www.glfw.org'
+description = """GLFW is an Open Source, multi-platform library for OpenGL,
+OpenGL ES and Vulkan development on the desktop"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c038d34200234d071fae9345bc455e4a8f2f544ab60150765d7704e08f3dac01']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('binutils', '2.39'),
+    ('pkgconf', '1.9.3'),
+]
+dependencies = [
+    ('Wayland', '1.22.0'),
+    ('X11', '20221110'),
+]
+
+# build both static and shared libraries
+configopts = [
+    '-DBUILD_SHARED_LIBS=OFF',
+    '-DBUILD_SHARED_LIBS=ON',
+]
+
+
+sanity_check_paths = {
+    'files': ['include/%(name)s/glfw3.h', 'include/%(name)s/glfw3native.h'] +
+             ['lib/libglfw3.a', 'lib/libglfw.%s' % SHLIB_EXT],
```

```suggestion
             ['lib/libglfw3.a', 'lib/libglfw.%s' % SHLIB_EXT, 'lib/pkgconfig/glfw3.pc'],
```

I see no reason to remove the check for glfw3.pc. It's still present, and important.